spec/unit/target/build_settings/pod_target_settings_spec.rb

```diff
@@ -189,19 +189,19 @@ class BuildSettings
             @pod_target.stubs(:platform).returns(Platform.new(:ios, '12.1'))
             generator = PodTargetSettings.new(@pod_target, nil, :configuration => :debug)
             hash = generator.generate.to_hash
-            hash['SYSTEM_FRAMEWORK_SEARCH_PATHS'].should.include '"$(PLATFORM_DIR)/Developer/usr/lib"'
-            hash['LIBRARY_SEARCH_PATHS'].should.include '"$(PLATFORM_DIR)/Developer/usr/lib"'
-            hash['SWIFT_INCLUDE_PATHS'].should.include '"$(PLATFORM_DIR)/Developer/usr/lib"'
+            hash['SYSTEM_FRAMEWORK_SEARCH_PATHS'].should.include '"$(inherited) $(PLATFORM_DIR)/Developer/Library/Frameworks"'
```

Comment:
can you make it `should ==` and be an exact match? I am surprised it doenst include `$(inherited)` already.